plugin-flex-ts-template-v2/src/feature-library/enhanced-crm-container/README.md

```diff
@@ -0,0 +1,17 @@
+# enhanced-crm-container
+
+This feature replaces the OOTB [CRMContainer](https://assets.flex.twilio.com/docs/releases/flex-ui/1.31.2/CRMContainer.html) with the aim of providing a more userfriendly iframe.
```

Comment:
Update link to https://assets.flex.twilio.com/docs/releases/flex-ui/2.0.0-beta.1/programmable-components/components/CRMContainer ?